pkg/gktest/result.go

```diff
@@ -42,4 +89,13 @@ type CaseResult struct {
 	// We don't need to distinguish between 1 and 2 - they are both treated as
 	// test failures.
 	Error error
+
+	// Runtime is the time it took for tests for this Constraint to run.
```

Comment:
This runtime is purely about evaluation time, right? No compilation, data loading, etc.?

Reply:
I've added some clarifying comments. This will be fully clarified later, especially in user documentation. This code and its comments will become stale literally in the next few days, so I don't want to spend time wordsmithing it.